MAUI/ImageEditor/getting-started.md

```diff
@@ -97,14 +97,18 @@ Refer to the following steps to add an image to the project:
 
 N> Image formats such as JPEG and PNG can be loaded to the image editor.

```

Comment:
is this step common to add types, if its particular move under
<img width="454" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/92789666/d8535077-18eb-49d8-8a2a-eb6e05990894">

Reply:
Moved a resource topic below